apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.tsx

```diff
@@ -47,11 +48,15 @@ function LaunchWpcomWelcomeTour() {
 	const { siteIntent, siteIntentFetched } = useSiteIntent();
 	const localeSlug = useLocale();
 	const editorType = getEditorType();
+	const showLaunchpad = getQueryArg( window.location.search, 'showLaunchpad' );
 
 	// Preload first card image (others preloaded after open state confirmed)
 	usePrefetchTourAssets( [ getTourSteps( localeSlug, false, false, null, siteIntent )[ 0 ] ] );
 
 	useEffect( () => {
+		if ( showLaunchpad ) {
```

Comment:
> While using the special flag showLaunchpad=true from a new post page (without iFrame), the Welcome Tour dialog should not show

Based on the description, could we limit this to the post editor? Otherwise, when the user lands on the site editor with `showLaunchpad=true`, the Welcome Tour won't show either but I think it's not expected.

Moreover, I assume the feature is also limited to the new writer flow. So I guess we still need to show the Welcome Tour if the user lands on the post editor with `showLaunchpad=true` from other places. Does it make sense?